components/inputs/input-text.js

```diff
@@ -312,6 +308,13 @@ class InputText extends FormElementMixin(SkeletonMixin(RtlMixin(LitElement))) {
 		return super.validity;
 	}
 
+	connectedCallback() {
+		super.connectedCallback();
+		if (this.ariaLabel) {
+			this.labelRequired = false;
```

Comment:
Maintaining support for the legacy `aria-label` attribute on this component means when it's set we need to _not_ throw a missing label exception.